### ISSUE TITLE
[Accessibility][Multiple Ways - Header] : Their should a shortcut method to access the header controls like copy, paste, cut etc.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2082]
**User Experience** 
The keyboard user and screen reader user will face difficulty to select and navigate back to the header control control to perform the action. 

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the chart and select any node
4. Try any shortcut such as delete button to delete the node or 'Ctrl + C' to copy.
4. Observe the issue.

**Actual Result:**
The basic shortcut's such as copy, cut, paste and delete can not performed by the shortcut keys.
Ex-
Ctrl + C to copy the node
Ctrl + X to cut the node
Ctrl + V to paste the copied node

**Expected:**
The basic functional shortcut should be enabled so that the user will have to navigate back again and again to perform the task.

**MAS Impacted:** [MAS2.4.5](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2fb4445-fc01-4830-892a-4b9efeda098a})

**Attachment:**
[MAS2.4.5_Shortcut keys for header's.zip](https://github.com/microsoft/BotFramework-Composer/files/4250082/MAS2.4.5_Shortcut.keys.for.header.s.zip)

**Comment by yeze322** [comment]
we've provided a config in visual editor's editorConfig.ts
![image](https://user-images.githubusercontent.com/8528761/78968887-3ec98280-7b38-11ea-891f-52e15077b2d5.png)
let's set `keyboardNodeEditing` to true

**Comment by cwhitten** [comment]
@alanlong9278 will update commands to the following (platform specific):

cmd(ctrl)-x = cut
cmd(ctrl)-c = copy
cmd(ctrl)-v = paste

@beyackle is working on a global `live-region` handler/store mechanism so it handles the scenario that when a user tries to paste on an invalid element, the alert will be appended to the announcements (globally) and the SR will announce appropriately. see #2050 

@alanlong9278 in the implementation for the key listeners on these commands, provide a comment that @beyackle can fill in once #2050 is addressed.

paste operation is only available when the focused node is the "+" element.

**Comment by beyackle** [comment]
The fix for #2050 is out and merged, so this should be all clear to proceed with the announcements on executing a command.

**Comment by alanlong9278** [comment]
@beyackle  the announcements on excuting commands will be added together in #2312

**Comment by ashish315** [comment]
As checked the issue is resolved. So, HCL is closing the issue.
